docs/book/storage/adapter.md

```diff
@@ -741,10 +741,12 @@ Name | Data Type | Default Value | Description
 ## Redis Adapter
 
 `Laminas\Cache\Storage\Adapter\Redis` stores cache items over the redis protocol
-using the PHP extension [redis](https://github.com/nicolasff/phpredis).
+using the PHP extension [redis](https://github.com/phpredis/phpredis).
```

Comment:
The extension is installed with "redis", but the official name seems to be "[PhpRedis](https://github.com/phpredis/phpredis#phpredis)".
```suggestion
using the PHP extension [PhpRedis](https://github.com/phpredis/phpredis).
```